eric_sse/entities.py

```
@@ -8,48 +8,40 @@
 
-    def get_queue(self, listener_id: str) -> Queue:
-        try:
-            return self.__queues[listener_id]
-        except KeyError:
-            raise InvalidListenerException(f"Invalid listener {listener_id}") from None
-
     def get_listener(self, listener_id: str) -> MessageQueueListener:
         try:
             return self.__listeners[listener_id]
         except KeyError:
             raise InvalidListenerException
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)